### PR TITLE
Feature: Test, Database and Routes folder translation

### PR DESCRIPTION
**Pull request by MercierCorentin**
In this PR I translated the tests, database and routes folder. Not a lot of changes but there still might be some mistakes.
Part of #242 resolution. See smae issue for plannification.
Edit: Sorry, I forgot the run the tests before pushing.

### CODE REVIEW

**Review by r0one — DISMISSED** [review]

**Comment by MercierCorentin** [comment]
@CdRom1 thanks for your review! Could you also translate your PR comments?

**Comment by r0one** [comment]
Translate my reviews and I'll review your translations :D

nah seriously I really don't have time for this now and it isn't affecting the quality of our code, but I'll try to remember to speak English on github ^^

Romain Maliach-Auguste

File: database/migrations/2010_10_12_000000_clear.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ public function up()
     }
 
     /**
-     * Supprime un dossier et tout son contenu.
+     * Deletes a folder and all its content.
```
Comment (NastuzziSamy) [inline database/migrations/2010_10_12_000000_clear.php]: Function comments are informative: Delete* 
ex: https://github.com/laravel/framework/blob/5.1/src/Illuminate/Routing/Controller.php#L42

File: database/migrations/2018_03_07_143023_create_groups_table.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ public function up()
             $table->collation = 'utf8_bin';
             $table->uuid('id')->primary();
             $table->uuid('user_id')->nullable();
-            // Correspond ici au créateur, pas à l'admin du groupe (qui peut changer)
+            // Corresponds here to the page creator, not to the group administrator (who can change)
```
Comment (NastuzziSamy) [inline database/migrations/2018_03_07_143023_create_groups_table.php]: Need a dot a the end :p

File: database/migrations/2018_03_07_143627_create_groups_members_table.php [hunk 3]
```diff
@@ -20,7 +20,7 @@ public function up()
             $table->uuid('user_id');
             $table->uuid('role_id')->nullable();
             $table->uuid('semester_id');
-            // On permet ici que le semestre soit égal à 0 pour ne convenir à aucun semestre
+            // We allow here the semester to be equal to 0 (Between two semesters for example).
```
Comment (NastuzziSamy) [inline database/migrations/2018_03_07_143627_create_groups_members_table.php:23]: Wrong translation here:
`We allow here the semester to be null so the membership could be for ever`

File: database/migrations/2018_07_02_181111_create_rooms_bookings_table.php [hunk 4]
```diff
@@ -19,7 +19,7 @@ public function up()
             $table->uuid('id')->primary();
             $table->uuid('room_id');
             $table->uuid('type_id')->nullable();
-            // Aucun type si on bloque les réservations via l'admin par ex
+            // No type if we stop the booking trough the admin interface for example
```
Comment (NastuzziSamy) [inline database/migrations/2018_07_02_181111_create_rooms_bookings_table.php]: through*

File: database/seeds/CalendarsTableSeeder.php [hunk 5]
```diff
@@ -141,7 +141,7 @@ public function run()
                 'description'	=> 'Calendrier des réunions du BDE-UTC',
                 'color'			=> '#FF0000',
                 'visibility'	=> 'private',
-            // Visible que par les membres
+            // Only visible by members.
```
Comment (NastuzziSamy) [inline database/seeds/CalendarsTableSeeder.php:144]: Can you add a tab before for the indentation ?

File: routes/admin.php [hunk 6]
```diff
@@ -10,23 +10,23 @@
 
 
 /**
- * Routes pour l'authentification admin.
+ * Admin authentification routes.
```
Comment (NastuzziSamy) [inline routes/admin.php]: authentication

File: routes/api/v1.php [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <?php
 /**
- * Défini les routes de l'api v1.
- * /!\ Les routes sont préfixées avec 'api/v1'.
+ * Defines v1 API routes.
+ * /!\ All Routes are prefixed with 'api/v1'.
```
Comment (NastuzziSamy) [inline routes/api/v1.php]: by*

File: routes/api/v1.php [hunk 8]
```diff
@@ -11,23 +11,23 @@
...
+ * Connections.
  */
 
 Route::get('login', 'Client\LoginController@index')->middleware('guest')->name('api/login');
 Route::get('logout', 'Client\LoginController@destroy')->middleware(Scopes::matchAnyUser())->name('api/logout');
 
 
 /*
- * Informations relatives à l'utlisateur.
+ * User related informations.
```
Comment (NastuzziSamy) [inline routes/api/v1.php]: information*

File: routes/api/v1.php [hunk 9]
```diff
@@ -124,7 +124,7 @@
         'users/{user_id}/services' => 'User\ServiceController',
 
     /*
-     * Routes `user` identiques à `users/{Auth::id()}`.
+     * Routes `user`identical to `users/{Auth::id()}`.
```
Comment (NastuzziSamy) [inline routes/api/v1.php]: user` iden*

File: routes/auth/cas.php [hunk 10]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Route de gestion du CAS-UTC.
+ * Cas Auth management routes.
```
Comment (NastuzziSamy) [inline routes/auth/cas.php]: auth*

**Review by NastuzziSamy — DISMISSED** [review]

File: database/migrations/2018_03_07_143627_create_groups_members_table.php (hunk 3 above)
Comment (huetremy) [reply database/migrations/2018_03_07_143627_create_groups_members_table.php]: To be null ;) (=nullable)

**Pull request merged**